scripts/pcm-dpc/covid-19/preprocess.py

```diff
@@ -0,0 +1,193 @@
+# Lint as: python3
+"""
+This script preprocesses the covid19 data from Italy Department of Civil
+Protection for importing into Data Commons.
+"""
+
+import pandas as pd
+
+class PcmDpc:
+    """download the csv files and preprocess it for importing into
+    DataCommons"""
+    def preprocess(self):
+        """clean and save the CSV file"""
+        self.data = pd.read_csv(self.csvpath)
+        assert 'note' in self.data.columns
+        self.data = self.data.drop(columns=['note'])
+        self._translate()
+        # drop the time, keep the date only
+        self.data['Date'] = self.data['Date'].str[0:10]
+        self.set_location() # prepocess the geo ids
+        self.data.to_csv(self.name + '.csv', index=False)
+
+    def generate_tmcf(self):
+        """generate the template mcf"""
+        geo_node = self.geo_template() # write the geo node to template mcf
+        TEMPLATE = ('Node: E:pcm-dpc->E{index}\n'
+                    'typeOf: dcs:StatVarObservation\n'
+                    'variableMeasured: dcs:{SVname}\n'
+                    'observationAbout: {geoNode}\n'
+                    'observationDate: C:pcm-dpc->Date\n'
+                    'value: C:pcm-dpc->{SVname}\n\n')
+        idx = 1
+        with open(self.name + '.tmcf', 'a') as f_out:
+            for statvar in self.stat_vars:
+                f_out.write(TEMPLATE.format_map({'index': idx,
+                    'SVname': statvar, 'geoNode': geo_node}))
+                idx += 1
+
+    def _translate(self):
+        """translate coloumn names from italian to english"""
+        it2en = {'data': 'Date', 'stato': 'State',
+                 'codice_regione':'RegionCode',
+                 'denominazione_regione': 'RegionName',
+                 'codice_provincia': 'ProvinceCode',
+                 'denominazione_provincia': 'ProvinceName',
+                 'sigla_provincia': 'ProvinceAbbreviation',
+                 'lat': 'Latitude', 'long': 'Longitude',
+                 'ricoverati_con_sintomi': 'Count_MedicalConditionIncident'
+                 '_COVID_19_PatientHospitalizedWithSymptoms',
```

Comment:
```suggestion
                 'ricoverati_con_sintomi': ('Count_MedicalConditionIncident'
                                                            '_COVID_19_PatientHospitalizedWithSymptoms'),
```
Using parentheses will help format the lines better.  Please apply here and elsewhere.